egs/librispeech/ASR/pruned_transducer_stateless2/train.py

```diff
@@ -973,6 +1010,7 @@ def scan_pessimistic_batches_for_oom(
                     f"Failing criterion: {criterion} "
                     f"(={crit_values[criterion]}) ..."
                 )
+                display_and_save_batch(batch, params=params, sp=sp)
```

Comment:
I would recommend put this line out of the `if`, since the error message is `RuntimeError: CUDA error: invalid configuration argument`. See https://github.com/k2-fsa/icefall/pull/318#issuecomment-1104683522 and https://github.com/k2-fsa/icefall/issues/247#issuecomment-1116206814

Reply:
> #247 (comment)

Thanks. Fixed in https://github.com/k2-fsa/icefall/pull/350